include/programs.h

```diff
@@ -58,14 +61,22 @@ class CommandLine {
 	                   const std::list<std::string_view>& post_args) const;
 	void Shift(unsigned int amount = 1);
 	uint16_t Get_arglength();
+	bool FindBoolArgument(const std::string& name, bool remove,
```

Comment:
I'd introduce two version of this for consistency with the other functions, and extra boolean args are best to be avoided:

- `FindRemoveBoolArgument(const std::string& name, char short_letter = 0)`
- `FindBoolArgument(const std::string& name, char short_letter = 0)`

Reply:
Fixed